### PR TITLE
Feature/v bodnarchuk/auth front

### PR DESCRIPTION
**Pull request by viktorzabila**

### CODE REVIEW

**Review by CodeGeneration-2020 — CHANGES_REQUESTED** [review]
Can you please consolidate spacings all over the app? For now it is a mess with curly braces
Whenever you declare prop - it should be:
```initialValues={{ title: '' }}```

Whenever you want to declare an object, it should be:

```const object = { title: '' }```

Whenever you do arguments destruction, it should be:

```({ handleChange, ...rest })```

File: mobile/src/components/EditTodo/EditTodo.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        {({handleChange, handleBlur, handleSubmit, values}) => (
+          <View>
+            <Input
+              title={'Title'}
+              onChangeText={handleChange('title')}
+              onBlur={handleBlur('title')}
+              value={values.title}
+            />
+            <ErrorMessage name='title'>{(msg) =>
+              <Text>{(msg)}</Text>}</ErrorMessage>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/EditTodo/EditTodo.tsx:59]: Move ```</ErrorMessage>``` to the next line
and remove redundant curly braces

File: mobile/src/components/EditTodo/EditTodo.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            <Input
+              title={'Description'}
+              multiline={'true'}
+              numberOfLines={8}
+              onChangeText={handleChange('description')}
+              onBlur={handleBlur('description')}
+              value={values.description}
+            />
+            <ErrorMessage name='description'>{(msg) =>
+              <Text>{msg}</Text>}</ErrorMessage>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/EditTodo/EditTodo.tsx:69]: Move ```</ErrorMessage>``` to the next line
and remove redundant curly braces

File: mobile/src/components/Register.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+          <View>
+            <Input
+              title={'Username'}
+              onChangeText={handleChange('username')}
+              onBlur={handleBlur('username')}
+              value={values.username}
+            />
+            <ErrorMessage name='username'>{(message) =>
+              <Text>
+                {message}</Text>}</ErrorMessage>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Register.tsx:47]: Move ```</ErrorMessage>``` to the next line
and remove redundant curly braces

File: mobile/src/components/Register.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            <Input
+              title={'Email'}
+              onChangeText={handleChange('email')}
+              onBlur={handleBlur('email')}
+              value={values.email}
+              keyboardType='email-address'
+            />
+            <ErrorMessage name='email'>{(message) =>
+              <Text>
+                {message}</Text>}</ErrorMessage>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Register.tsx:57]: Move ```</ErrorMessage>``` to the next line
and remove redundant curly braces

File: mobile/src/components/Register.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            <Input
+              title={'Password'}
+              onChangeText={handleChange('password')}
+              onBlur={handleBlur('password')}
+              value={values.password}
+              secureTextEntry={'true'}
+            />
+            <ErrorMessage name='password'>{(message) =>
+              <Text>{message}
+              </Text>}</ErrorMessage>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Register.tsx:67]: Move ```</ErrorMessage>``` to the next line
and remove redundant curly braces

File: mobile/src/components/Register.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+            <Input
+              title={'Verify password'}
+              onChangeText={handleChange('verifyPassword')}
+              onBlur={handleBlur('verifyPassword')}
+              value={values.verifyPassword}
+              secureTextEntry={'true'}
+            />
+            <Text>
+              {!match ? 'Passwords do not match' : ''}</Text>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Register.tsx:76]: move ```</Text>``` to the next line

File: mobile/src/components/Todo/Todo.js [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+        style={styles.button}
+        onPress={onPressCreate}
+      >
+        <Text>Create new Todo</Text>
+      </TouchableOpacity>
+
+      <FlatList
+        style={styles.todosWrapper}
+        data={todos}
+        renderItem={({item}) =>
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Todo/Todo.js:77]:
```
renderItem={({ item }) => (
        <TodoItem
            todoItem={item}
            onToDoPressEdit={onPressEdit}
            onToDoPressDelete={onPressDelete}
          />
)}

File: mobile/src/components/Todo/TodoItem.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+import {FontAwesome, MaterialCommunityIcons} from '@expo/vector-icons';
+export default function TodoItem({todoItem,
+  onToDoPressEdit, onToDoPressDelete}) {
```
Comment (CodeGeneration-2020) [inline mobile/src/components/Todo/TodoItem.js:5]: mess with spacings here

**Review by CodeGeneration-2020 — CHANGES_REQUESTED** [review]

**Comment by viktorzabila** [comment]
Здравствуйте, не могу зайти в Slack, извините что не был на коле, у меня
отключили светло.

ср, 15 сент. 2021 г., 09:37 Code Generation ***@***.***>:

**Pull request closed without merging**